app/views/shared/_navigation_bar.html.erb

```diff
@@ -15,14 +15,14 @@
             <%= link_to 'Journalize', journal_path, class: "nav-link"%>
           </li>
           <li class="nav-item <%= active_class(accounts_path) %>">
-            <%= link_to 'Chart of Accounts', accounts_path, class: "nav-link"%>
+            <%= link_to 'Accounts', accounts_path, class: "nav-link"%>
           </li>
```

Comment:
He might actually want it to be called "Chart of Accounts"  I think the chart of accounts has all accounts and then "Accounts" would have only the active accounts